package.json

```diff
@@ -20,7 +20,20 @@
     ],
     "main": "./out/extension",
     "contributes": {
-        "commands": [],
+        "commands": [
+            {
+                "command": "railsI18n.copyYamlKey",
+                "title": "Copy Key"
```

Comment:
The command name as it is seems very generic, would it be better if we changed it to "Copy I18n Key"?